docusaurus.config.js

```diff
@@ -111,6 +111,11 @@ const config = {
             type: "localeDropdown",
             position: "right",
           },
+          {
+            href: 'https://rancher.com',
```

Comment:
```suggestion
            href: 'https://www.rancher.com',
```

Linking to rancher.com without the www negatively impacts our SEO score (per ContentKing) as it's a redirect.

Reply:
Just updated, thanks for the review Billy!